skimage/feature/texture.py

```diff
@@ -20,18 +20,21 @@ def greycomatrix(image, distances, angles, levels=256, symmetric=False,
 
     Parameters
     ----------
-    image : array_like of uint8
-        Integer typed input image. The image will be cast to uint8, so
-        the maximum value must be less than 256.
+    image : array_like of uint8 or uint16.
+        Integer typed input image. Only positive valued images are supported. 
+        If type is uint16, the argument `levels` needs to be set.
```

Comment:
Ok. Thanks a lot for your comments. I modified the doc strings accordingly. 

One question: do I also need to include new tests?